library/crates/Cargo.toml

```diff
@@ -10,12 +10,10 @@ antlr-rust = "=0.3.0-beta"
 axum = "=0.5.15"
 chrono = "=0.4.22"
 crossbeam = "=0.8.2"
-core-foundation-sys = "=0.8.3" # transitive dependency of 'iana_time_zone', included to patch cfg('unix') issue (#385)
```

Comment:
We included these dependencies explicitly to patch the issue of them not being installed on an M1 Mac, but now we've actually fixed the root cause of the issue.